k9mail/src/main/java/com/fsck/k9/mailstore/LocalMessage.java

```diff
@@ -315,6 +316,39 @@ public Void doDbWork(final SQLiteDatabase db) throws WrappedException, Unavailab
         localStore.notifyChange();
     }
 
+    public void debugClearLocalData() throws MessagingException {
+        if (!K9.DEBUG) {
```

Comment:
heh, if you had only put one comment I would probably have looked for others myself :P